### DISCUSSION TITLE
Berkeley 电子工程与计算机科学 | HackWay技术学习路线

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#42] [Announcements]
# Berkeley 电子工程与计算机科学 | HackWay技术学习路线

作为四大CS名校之一，伯克利在计算机领域的研究对计算机科学的实际应用产生过重大影响，比如和乔布斯创建苹果的天才程序员Steve Wozniak，伯克利UNIX、INGRES数据库系统、精简指令集RISC、计算复杂度理论等等。而且伯克利CS课程的资料非常新而且全，非常推荐大家来学习。

https://hackway.org/docs/cs/path/cmu

**Comment by auravan** [comment]
最好不要同时选那几个都是重量级。仙之人兮列如麻。
